src/platform/graphics/constants.js

```diff
@@ -655,12 +782,36 @@ export const pixelFormatInfo = new Map([
     [PIXELFORMAT_PVRTC_4BPP_RGBA_1, { name: 'PVRTC_4BPP_RGBA_1', blockSize: 8 }],
     [PIXELFORMAT_ASTC_4x4, { name: 'ASTC_4x4', blockSize: 16 }],
     [PIXELFORMAT_ATC_RGB, { name: 'ATC_RGB', blockSize: 8 }],
-    [PIXELFORMAT_ATC_RGBA, { name: 'ATC_RGBA', blockSize: 16 }]
+    [PIXELFORMAT_ATC_RGBA, { name: 'ATC_RGBA', blockSize: 16 }],
+
+    // uncompressed integer formats (WebGL2 Only)
+    [PIXELFORMAT_R8I,       { name: 'R8I', size: 1, channelSize: 1 }],
```

Comment:
`channelSize` does not seem to be used anywhere else apart from `isIntegerPixelFormat` function. Should we just add 'int` boolean flag instead?